Resources/Prototypes/GameRules/unknown_shuttles.yml

```
@@ -272,3 +273,16 @@
   - type: LoadMapRule
     preloadedGrid: Spacebus
 
+# Sunrise-Start
+- type: entity
+  id: ERTShuttleGammaMedium
+  parent: BaseUnknownShuttleRule
+  components:
+  - type: StationEvent
+    startAnnouncement: null
+    startAudio: /Audio/_Sunrise/Announcements/sunrise_ert_accepted.ogg
+    duration: -0 # Если изменить будет двоиться
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)